core/cli/src/rc-file.ts

```diff
@@ -44,6 +46,27 @@ export async function loadToolKitRC(logger: Logger, root: string, isAppRoot: boo
   }
 
   const config: RawRCFile = result.config
+
+  // if a toolkitrc contains a non-empty options field, but not options.{plugins,tasks,hooks},
+  // assume it's an old-style, plugins-only options field.
+  if (
+    config.options &&
+    Object.keys(config.options).length > 0 &&
+    !(config.options.plugins || config.options.tasks || config.options.hooks)
+  ) {
+    config.options = {
+      plugins: config.options as { [id: string]: Record<string, unknown> }
```

Comment:
**issue**: will this work for options that are now task options instead of plugin options?

Reply:
it will not. there's not going to be an easy way to automatically handle those in general. this will let those options fall through to the plugin option validator, which it looks like will just skip plugins that don't have a schema. what i could do is in #570 keep the schemas i deleted but change them all to `z.never()`, which should fail validation with a hopefully useful error message?

Reply:
yeah, that sounds like a good compromise

Reply:
done in https://github.com/Financial-Times/dotcom-tool-kit/pull/570/commits/cb98a854a582fe6cb76a5c57156ff5101afcb1bf